test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,35 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void iaAnyNull() throws Exception {
+        // empty list
+        assertContainsNoNull();
+
+        // all objects are not null
+        assertContainsNoNull("abc", "ab", "a");
+        assertContainsNoNull(1, 2);
+
+        // only one null
+        assertContainsNull(null, 1);
+        assertContainsNull("", null);
+        assertContainsNull("", null, "abc");
+
+        // some are null
+        assertContainsNull("abc", null, "abc", null, null);
+        assertContainsNull("abc", "", "ABC", null);
+        assertContainsNull(null, "", "abc", null, "a", "abc");
+
+        // all are null
+        assertContainsNull(null, null);
+    }
+
+    private void assertContainsNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
```

Comment:
Not necessary to abstract the `assertTrue(Utils.isAnyNull(...))` as separate method, because asserting the result of the method itself is fine.

Although existing test `assertAreUnique` does that, but that is to prevent `Arrays.asList(...)` from flying everywhere.